frontend/app/components/input-checkboxes.tsx

```diff
@@ -0,0 +1,72 @@
+import type { ComponentProps, ReactNode } from 'react';
+
+import { InputCheckbox } from './input-checkbox';
+import { InputError } from './input-error';
+import { InputHelp } from './input-help';
```

Comment:
```suggestion
import { InputCheckbox } from '~/components/input-checkbox';
import { InputError } from '~/components/input-error';
import { InputHelp } from '~/components/input-help';
```

Reply:
Implemented in commit dec2ceef9218bed7957453837c8f49c7cb1d2692.